evennia/contrib/actions/README.md

```diff
@@ -0,0 +1,372 @@
+ 
+
+#### stop
+>    stop all of your ongoing and/or queued actions
+>
+>    Usage:
+>        stop [<ongoing/queue>]
+>
+>    Simply using "stop" will stop both ongoing and queued actions.
+ 
+
+#### done 
+>    ends the character's turn during turn-based situations
+>
+>    Usage:
+>        done
+>        do
+
+
+#### queue
+>    displays the character's action queue
+>
+>    Usage:
+>        queue
+>        que
+
+
+The commands in the ActionDebugCmdSet are meant for wizards, immortals and the superuser, and include the following:
+
+#### @@actslow
+>    performs an action with no bodyparts over 30 seconds
+
+
+#### @@actfast
+>    performs an action with no bodyparts over 10 seconds
+
+
+#### @@actlarm
+>    performs an action with the 'left arm' bodypart over 20 seconds
+
+
+#### @@actlarm
+>    performs an action with the 'right arm' bodypart over 20 seconds
+
+
+#### @@actfriend
+>    performs an action towards the target, does not invoke turn-based mode
+>
+>    Usage:
+>        @@actfriend <target>
+
+
+#### @@actfoe
+>    performs an action towards the target, invokes turn-based mode
+>
+>    Usage:
+>        @@actfoe <target>
+
+
+#### @@actsetup
+>    calls evennia.contrib.actions.setup(). If you specify the argument "over",
+>    the override flag will be on.
+>
+>    Usage:
+>        @@actsetup 
+>        @@actsetup over
+
+#### @@debugmsg
+    Enables / disables the debug messages that can be found in various parts
+    of the actions system's methods. Only those who have this setting enabled
+    can receive the messages.
+
+    Usage:
+        @@debug [<on/off>]
+
+    The use of "@@debug" on its own enables debug messages.
+
+
+## Limitations of the system
+
+* For the time being, only one object may be the target of a given action.
+* The various messages sent to the players regarding actions that are being performed all require the action performer's name to be present at the start of the message, rather than in the middle of it.
+* In turn-based mode, PCs get 60 seconds to complete their turn, while NPCs get 5. There is no way to customize this at present.
+
+All of these issues can be resolved, and will be resolved if people who are planning to use the system contact the maintainer, Andrei Pambuccian, about it. My github name is andrei-pmbcn.
+
+Other limitations that are unlikely to be resolved include the following:
+
+* There needs to be at most one exit pointing to a given room B in any room A if B also has an exit to A. If two exits in A point to the same room B, which has an exit to A, then when moving into A after having left B, the distance used to calculate the movement action's duration will be the distance value assigned to the first of the exits from A to B, whichever of the two exits that happens to be. Confused? Just remember that when two rooms have more than one exit to each other, and the distances of these exits are different, the action system will pick one of these distances, which may be the wrong one.
```

Comment:
In order to be reproducible, I guess it could automatically go for the shortest of the multiple possible exit routes?